backend/cache_utils.py

```
@@ -86,3 +86,37 @@ def invalidate_tab_feeds_cache(tab_id, invalidate_tabs=True):
+            invalidate_tabs_cache()
+        return
+
+    # Create keys
+    keys = [get_tab_version_key(t_id) for t_id in tab_ids_list]
+
+    # Fetch current versions in a single round-trip
+    current_versions = cache.get_many(*keys)
+
+    # Calculate new versions and create updates dictionary
+    updates = {}
+    for key, current_v in zip(keys, current_versions):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)